build/release.sh

```diff
@@ -36,31 +44,49 @@ export BUILD_USER="$git_user"
 export BUILD_DATE=$( date +%Y%m%d ) # Release date is only to day-granularity
 export VERBOSE=true
 
-# Build the release binary with libpfm4 for docker container
-export GO_FLAGS="-tags=libpfm,netgo"
-build/build.sh
-
 # Build the docker image
 echo ">> building cadvisor docker image"
-gcr_tag="gcr.io/cadvisor/cadvisor:$VERSION"
-docker build -t $gcr_tag -f deploy/Dockerfile .
+image_name=${IMAGE_NAME:-"gcr.io/cadvisor/cadvisor"}
+final_image="$image_name:${VERSION}"
 
-# Build the release binary without libpfm4 to not require libpfm4 in runtime environment
-unset GO_FLAGS
-build/build.sh
+docker buildx inspect cadvisor-builder > /dev/null \
+|| docker buildx create --name cadvisor-builder --use
```

Comment:
I made this builder persistent, it helps with local testing because the unchanged parts of the image are cached. Let me know if you want me to return to deleting the builder afterwards.

Reply:
sounds good!